python/tk_multi_workfiles/file_form_base.py

```diff
@@ -231,6 +231,10 @@ def _on_refresh_triggered(self, checked=False):
 
         :param checked:    True if the refresh action is checked - ignored
         """
+        app = sgtk.platform.current_bundle()
```

Comment:
This will sync the path cache every single time the user hit's F5 or Command-R/Ctrl-R